src/index.js

```diff
@@ -36,4 +37,39 @@ function link(e) {
     //Links code
 }
 
+// -----Projects Section-----
+const projectsButton = document.querySelectorAll('.projects__filter-item');
+const projects = document.querySelectorAll('.projects__project');
+
+[...projectsButton].forEach(function (element) {
+    element.addEventListener('click', portfolio);
+});
```

Comment:
Let's use ES6. 
```js
[...projectsButton].forEach(e => {
    e.addEventListener('click', portfolio);
});
```

Reply:
I'm not sure "for of" will work correctly everywhere

Reply:
I didn't talk about `for of`, `forEach` is totally fine. Look at my example above.